android/app/src/main/java/io/mosip/residentapp/MyPackage.java

```diff
@@ -0,0 +1,22 @@
+package io.mosip.residentapp;
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
+import com.reactlibrary.RNPixelpassModule;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MyPackage implements ReactPackage {
+
+    @Override
+    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
+        return Arrays.asList(new RNPixelpassModule(reactContext));
```

Comment:
We can add all modules here? Like VciClientModule, tuvali?

Reply:
yes